api/util.py

```diff
@@ -15,9 +12,15 @@
 import mimetypes
 import dateutil.parser
 import tempdir as tempfile
+import logging
 
 import scitran.data
 
+log_fmt = '%(asctime)s %(message)s'
+datefmt = '%Y-%m-%d %H:%M:%S'
+log = logging.getLogger('scitran.api')
+logging.basicConfig(format=log_fmt, datefmt=datefmt, level=logging.INFO)
+
```

Comment:
Ideally, we'd only have one definition of our logging style. We currently have this one and one in ```api.wsgi```. ```util.py``` is probably the right place, but I prefer the definition we have in ```api.wsgi```.